memcached/plan.sh

```diff
@@ -11,6 +11,7 @@ pkg_deps=(
   core/glibc
   core/cyrus-sasl
   core/libevent
+  core/openssl11
```

Comment:
Note,  `core/cyrus-sasl` deps on `core/openssl`, and we don't provide any guards against this behavior, or guarantees which will be loaded.

Reply:
Good catch, I can move core/cyrus-sasl to openssl11 but...uff, the dep tree might mean we pull in a lot.  One option is I can just do this in the chef origin.

Reply:
I wonder if this points to a missing feature.  When two packages provide implementations of the same shared library.  Perhaps we should have a pkg_provides_lib variable that we could use to generate conflicts in this case.

Reply:
I was thinking the same thing. This has also come up in the past with `gcc` and `gcc-libs`.  I don't feel it's reasonable for core plans contributors to have to be aware of this case, and we should be able to provide sufficient guards in our build process. 

However, there might also be some documentation and policies needed around `core` only using latest as a dependency, in order to keep the graph sane.